error when executing make l4t-32.3.1-jax

Hi @idavis ,

Here is a first error I encountered.

When running:

```
jetson-containers$ sudo make l4t-32.3.1-jax
```

I get
```
Step 11/32 : RUN apt-get update && apt-get install -y --no-install-recommends     bzip2     lbzip2     ca-certificates     gnupg     sudo
 ---> Running in b3bd7493802f
standard_init_linux.go:211: exec user process caused "no such file or directory"
```

Hi @idavis ,

- OS: Ubuntu 18.04.4 LTS 
- runtime: Docker version 19.03.5
- Whole build log follows below

PS
`make l4t-32.2.1-jax` gives me the same error; I tried to run it on jetson-containers before #27, and it worked fine.


```
jetson-containers$ sudo make l4t-32.3.1-jax
make -C jetson-containers/docker/l4t 32.3.1-jax
make[1]: Entering directory 'jetson-containers/docker/l4t'
docker build --squash \
				--build-arg VERSION_ID="bionic-20200112" \
				--build-arg DEPENDENCIES_IMAGE=l4t:32.3.1-jax-jetpack-4.3-deps \
				-t l4t:32.3.1-jax \
				- < jetson-containers/docker/l4t/32.3.1/jax/Dockerfile
Sending build context to Docker daemon   5.12kB
Step 1/32 : ARG VERSION_ID
Step 2/32 : ARG DEPENDENCIES_IMAGE
Step 3/32 : ARG IMAGE_NAME
Step 4/32 : FROM ${DEPENDENCIES_IMAGE} as dependencies
 ---> b8f3fbbbeda1
Step 5/32 : ARG VERSION_ID
 ---> Using cache
 ---> c5b8edeaa708
Step 6/32 : FROM ubuntu:${VERSION_ID} as qemu
 ---> ccc6e87d482b
Step 7/32 : RUN apt-get update && apt-get install -y --no-install-recommends qemu-user-static &&     apt-get clean &&     rm -rf /var/lib/apt/lists/*
 ---> Using cache
 ---> fd0919abbd41
Step 8/32 : ARG VERSION_ID
 ---> Using cache
 ---> 2cf05037f784
Step 9/32 : FROM arm64v8/ubuntu:${VERSION_ID}
 ---> db472617b3d1
Step 10/32 : LABEL maintainer "Ian Davis"
 ---> Using cache
 ---> 04a7b59627cc
Step 11/32 : RUN apt-get update && apt-get install -y --no-install-recommends     bzip2     lbzip2     ca-certificates     gnupg     sudo
 ---> Running in 28421b1c10e9
standard_init_linux.go:211: exec user process caused "no such file or directory"
The command '/bin/sh -c apt-get update && apt-get install -y --no-install-recommends     bzip2     lbzip2     ca-certificates     gnupg     sudo' returned a non-zero code: 1
Makefile:29: recipe for target '32.3.1-jax' failed
make[1]: *** [32.3.1-jax] Error 1
make[1]: Leaving directory 'jetson-containers/docker/l4t'
Makefile:50: recipe for target 'l4t-32.3.1-jax' failed
make: *** [l4t-32.3.1-jax] Error 2
```

@idavis 

Thanks, that part passes now. I still get an error:

```
Pre-installing bootloader package, skip flashing
Setting up nvidia-l4t-multimedia-utils (32.3.1-20191209230245) ...
Setting up nvidia-l4t-3d-core (32.3.1-20191209230245) ...
Setting up libpangocairo-1.0-0:arm64 (1.40.14-1ubuntu0.1) ...
Setting up nvidia-l4t-cuda (32.3.1-20191209230245) ...
Setting up nvidia-l4t-multimedia (32.3.1-20191209230245) ...
Setting up nvidia-l4t-camera (32.3.1-20191209230245) ...
Setting up nvidia-l4t-graphics-demos (32.3.1-20191209230245) ...
Setting up nvidia-l4t-gstreamer (32.3.1-20191209230245) ...
Processing triggers for libc-bin (2.27-3ubuntu1) ...
Processing triggers for nvidia-l4t-kernel (4.9.140-tegra-32.3.1-20191209230245) ...
/
Removing QEMU binary from rootfs
Removing stashed Debian packages from rootfs
/Linux_for_Tegra/nv_tegra/nv-apply-debs.sh: line 216: /bin/rm: No such file or directory
The command '/bin/sh -c ./Linux_for_Tegra/apply_binaries.sh -r /' returned a non-zero code: 127
Makefile:29: recipe for target '32.3.1-jax' failed
make[1]: *** [32.3.1-jax] Error 127
make[1]: Leaving directory 'jetson-containers/docker/l4t'
Makefile:50: recipe for target 'l4t-32.3.1-jax' failed
make: *** [l4t-32.3.1-jax] Error 2
```

--- a/docker/l4t/32.3.1/jax-8gb/Dockerfile
+++ b/docker/l4t/32.3.1/jax-8gb/Dockerfile
@@ -42,6 +42,7 @@ RUN sed -i '/.*LC_ALL=C APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=1 chroot . apt-key 
 RUN sed -i '/.*umount ${L4T_ROOTFS_DIR}\/proc.*/c\#umount ${L4T_ROOTFS_DIR}\/proc' ./Linux_for_Tegra/nv_tegra/nv-apply-debs.sh
 RUN sed -i '/.*	LC_ALL=C chroot . dpkg -i --path-include=\"\/usr\/share\/doc\/\*\" \"${pre_deb_list\[\@\]}\".*/c\	apt install -y --no-install-recommends \"${pre_deb_list[@]}\"' ./Linux_for_Tegra/nv_tegra/nv-apply-debs.sh
 RUN sed -i '/.*LC_ALL=C chroot . dpkg -i --path-include=\"\/usr\/share\/doc\/\*\" \"${deb_list\[\@\]}\".*/c\apt install -y --no-install-recommends \"${deb_list[@]}\"' ./Linux_for_Tegra/nv_tegra/nv-apply-debs.sh
+RUN sed -i '/.*rm -f "${L4T_ROOTFS_DIR}\/usr\/bin\/qemu-aarch64-static\".*/c\#rm -f "${L4T_ROOTFS_DIR}\/usr\/bin\/qemu-aarch64-static\"' ./Linux_for_Tegra/nv_tegra/nv-apply-debs.sh
 
 # In 32.3.1, this will use deb packages instead of the archive overlay
 RUN ./Linux_for_Tegra/apply_binaries.sh -r /

--- a/docker/l4t/32.3.1/jax/Dockerfile
+++ b/docker/l4t/32.3.1/jax/Dockerfile
@@ -42,6 +42,7 @@ RUN sed -i '/.*LC_ALL=C APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=1 chroot . apt-key 
 RUN sed -i '/.*umount ${L4T_ROOTFS_DIR}\/proc.*/c\#umount ${L4T_ROOTFS_DIR}\/proc' ./Linux_for_Tegra/nv_tegra/nv-apply-debs.sh
 RUN sed -i '/.*	LC_ALL=C chroot . dpkg -i --path-include=\"\/usr\/share\/doc\/\*\" \"${pre_deb_list\[\@\]}\".*/c\	apt install -y --no-install-recommends \"${pre_deb_list[@]}\"' ./Linux_for_Tegra/nv_tegra/nv-apply-debs.sh
 RUN sed -i '/.*LC_ALL=C chroot . dpkg -i --path-include=\"\/usr\/share\/doc\/\*\" \"${deb_list\[\@\]}\".*/c\apt install -y --no-install-recommends \"${deb_list[@]}\"' ./Linux_for_Tegra/nv_tegra/nv-apply-debs.sh
+RUN sed -i '/.*rm -f "${L4T_ROOTFS_DIR}\/usr\/bin\/qemu-aarch64-static\".*/c\#rm -f "${L4T_ROOTFS_DIR}\/usr\/bin\/qemu-aarch64-static\"' ./Linux_for_Tegra/nv_tegra/nv-apply-debs.sh
 
 # In 32.3.1, this will use deb packages instead of the archive overlay
 RUN ./Linux_for_Tegra/apply_binaries.sh -r /

--- a/docker/l4t/32.3.1/nano-dev/Dockerfile
+++ b/docker/l4t/32.3.1/nano-dev/Dockerfile
@@ -42,6 +42,7 @@ RUN sed -i '/.*LC_ALL=C APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=1 chroot . apt-key 
 RUN sed -i '/.*umount ${L4T_ROOTFS_DIR}\/proc.*/c\#umount ${L4T_ROOTFS_DIR}\/proc' ./Linux_for_Tegra/nv_tegra/nv-apply-debs.sh
 RUN sed -i '/.*	LC_ALL=C chroot . dpkg -i --path-include=\"\/usr\/share\/doc\/\*\" \"${pre_deb_list\[\@\]}\".*/c\	apt install -y --no-install-recommends \"${pre_deb_list[@]}\"' ./Linux_for_Tegra/nv_tegra/nv-apply-debs.sh
 RUN sed -i '/.*LC_ALL=C chroot . dpkg -i --path-include=\"\/usr\/share\/doc\/\*\" \"${deb_list\[\@\]}\".*/c\apt install -y --no-install-recommends \"${deb_list[@]}\"' ./Linux_for_Tegra/nv_tegra/nv-apply-debs.sh
+RUN sed -i '/.*rm -f "${L4T_ROOTFS_DIR}\/usr\/bin\/qemu-aarch64-static\".*/c\#rm -f "${L4T_ROOTFS_DIR}\/usr\/bin\/qemu-aarch64-static\"' ./Linux_for_Tegra/nv_tegra/nv-apply-debs.sh
 
 # In 32.3.1, this will use deb packages instead of the archive overlay
 RUN ./Linux_for_Tegra/apply_binaries.sh -r /

--- a/docker/l4t/32.3.1/nano/Dockerfile
+++ b/docker/l4t/32.3.1/nano/Dockerfile
@@ -42,6 +42,7 @@ RUN sed -i '/.*LC_ALL=C APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=1 chroot . apt-key 
 RUN sed -i '/.*umount ${L4T_ROOTFS_DIR}\/proc.*/c\#umount ${L4T_ROOTFS_DIR}\/proc' ./Linux_for_Tegra/nv_tegra/nv-apply-debs.sh
 RUN sed -i '/.*	LC_ALL=C chroot . dpkg -i --path-include=\"\/usr\/share\/doc\/\*\" \"${pre_deb_list\[\@\]}\".*/c\	apt install -y --no-install-recommends \"${pre_deb_list[@]}\"' ./Linux_for_Tegra/nv_tegra/nv-apply-debs.sh
 RUN sed -i '/.*LC_ALL=C chroot . dpkg -i --path-include=\"\/usr\/share\/doc\/\*\" \"${deb_list\[\@\]}\".*/c\apt install -y --no-install-recommends \"${deb_list[@]}\"' ./Linux_for_Tegra/nv_tegra/nv-apply-debs.sh
+RUN sed -i '/.*rm -f "${L4T_ROOTFS_DIR}\/usr\/bin\/qemu-aarch64-static\".*/c\#rm -f "${L4T_ROOTFS_DIR}\/usr\/bin\/qemu-aarch64-static\"' ./Linux_for_Tegra/nv_tegra/nv-apply-debs.sh
 
 # In 32.3.1, this will use deb packages instead of the archive overlay
 RUN ./Linux_for_Tegra/apply_binaries.sh -r /

--- a/docker/l4t/32.3.1/tx1/Dockerfile
+++ b/docker/l4t/32.3.1/tx1/Dockerfile
@@ -42,6 +42,7 @@ RUN sed -i '/.*LC_ALL=C APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=1 chroot . apt-key 
 RUN sed -i '/.*umount ${L4T_ROOTFS_DIR}\/proc.*/c\#umount ${L4T_ROOTFS_DIR}\/proc' ./Linux_for_Tegra/nv_tegra/nv-apply-debs.sh
 RUN sed -i '/.*	LC_ALL=C chroot . dpkg -i --path-include=\"\/usr\/share\/doc\/\*\" \"${pre_deb_list\[\@\]}\".*/c\	apt install -y --no-install-recommends \"${pre_deb_list[@]}\"' ./Linux_for_Tegra/nv_tegra/nv-apply-debs.sh
 RUN sed -i '/.*LC_ALL=C chroot . dpkg -i --path-include=\"\/usr\/share\/doc\/\*\" \"${deb_list\[\@\]}\".*/c\apt install -y --no-install-recommends \"${deb_list[@]}\"' ./Linux_for_Tegra/nv_tegra/nv-apply-debs.sh
+RUN sed -i '/.*rm -f "${L4T_ROOTFS_DIR}\/usr\/bin\/qemu-aarch64-static\".*/c\#rm -f "${L4T_ROOTFS_DIR}\/usr\/bin\/qemu-aarch64-static\"' ./Linux_for_Tegra/nv_tegra/nv-apply-debs.sh
 
 # In 32.3.1, this will use deb packages instead of the archive overlay
 RUN ./Linux_for_Tegra/apply_binaries.sh -r /

--- a/docker/l4t/32.3.1/tx2-4gb/Dockerfile
+++ b/docker/l4t/32.3.1/tx2-4gb/Dockerfile
@@ -42,6 +42,7 @@ RUN sed -i '/.*LC_ALL=C APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=1 chroot . apt-key 
 RUN sed -i '/.*umount ${L4T_ROOTFS_DIR}\/proc.*/c\#umount ${L4T_ROOTFS_DIR}\/proc' ./Linux_for_Tegra/nv_tegra/nv-apply-debs.sh
 RUN sed -i '/.*	LC_ALL=C chroot . dpkg -i --path-include=\"\/usr\/share\/doc\/\*\" \"${pre_deb_list\[\@\]}\".*/c\	apt install -y --no-install-recommends \"${pre_deb_list[@]}\"' ./Linux_for_Tegra/nv_tegra/nv-apply-debs.sh
 RUN sed -i '/.*LC_ALL=C chroot . dpkg -i --path-include=\"\/usr\/share\/doc\/\*\" \"${deb_list\[\@\]}\".*/c\apt install -y --no-install-recommends \"${deb_list[@]}\"' ./Linux_for_Tegra/nv_tegra/nv-apply-debs.sh
+RUN sed -i '/.*rm -f "${L4T_ROOTFS_DIR}\/usr\/bin\/qemu-aarch64-static\".*/c\#rm -f "${L4T_ROOTFS_DIR}\/usr\/bin\/qemu-aarch64-static\"' ./Linux_for_Tegra/nv_tegra/nv-apply-debs.sh
 
 # In 32.3.1, this will use deb packages instead of the archive overlay
 RUN ./Linux_for_Tegra/apply_binaries.sh -r /

--- a/docker/l4t/32.3.1/tx2/Dockerfile
+++ b/docker/l4t/32.3.1/tx2/Dockerfile
@@ -42,6 +42,7 @@ RUN sed -i '/.*LC_ALL=C APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=1 chroot . apt-key 
 RUN sed -i '/.*umount ${L4T_ROOTFS_DIR}\/proc.*/c\#umount ${L4T_ROOTFS_DIR}\/proc' ./Linux_for_Tegra/nv_tegra/nv-apply-debs.sh
 RUN sed -i '/.*	LC_ALL=C chroot . dpkg -i --path-include=\"\/usr\/share\/doc\/\*\" \"${pre_deb_list\[\@\]}\".*/c\	apt install -y --no-install-recommends \"${pre_deb_list[@]}\"' ./Linux_for_Tegra/nv_tegra/nv-apply-debs.sh
 RUN sed -i '/.*LC_ALL=C chroot . dpkg -i --path-include=\"\/usr\/share\/doc\/\*\" \"${deb_list\[\@\]}\".*/c\apt install -y --no-install-recommends \"${deb_list[@]}\"' ./Linux_for_Tegra/nv_tegra/nv-apply-debs.sh
+RUN sed -i '/.*rm -f "${L4T_ROOTFS_DIR}\/usr\/bin\/qemu-aarch64-static\".*/c\#rm -f "${L4T_ROOTFS_DIR}\/usr\/bin\/qemu-aarch64-static\"' ./Linux_for_Tegra/nv_tegra/nv-apply-debs.sh
 
 # In 32.3.1, this will use deb packages instead of the archive overlay
 RUN ./Linux_for_Tegra/apply_binaries.sh -r /

--- a/docker/l4t/32.3.1/tx2i/Dockerfile
+++ b/docker/l4t/32.3.1/tx2i/Dockerfile
@@ -42,6 +42,7 @@ RUN sed -i '/.*LC_ALL=C APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=1 chroot . apt-key 
 RUN sed -i '/.*umount ${L4T_ROOTFS_DIR}\/proc.*/c\#umount ${L4T_ROOTFS_DIR}\/proc' ./Linux_for_Tegra/nv_tegra/nv-apply-debs.sh
 RUN sed -i '/.*	LC_ALL=C chroot . dpkg -i --path-include=\"\/usr\/share\/doc\/\*\" \"${pre_deb_list\[\@\]}\".*/c\	apt install -y --no-install-recommends \"${pre_deb_list[@]}\"' ./Linux_for_Tegra/nv_tegra/nv-apply-debs.sh
 RUN sed -i '/.*LC_ALL=C chroot . dpkg -i --path-include=\"\/usr\/share\/doc\/\*\" \"${deb_list\[\@\]}\".*/c\apt install -y --no-install-recommends \"${deb_list[@]}\"' ./Linux_for_Tegra/nv_tegra/nv-apply-debs.sh
+RUN sed -i '/.*rm -f "${L4T_ROOTFS_DIR}\/usr\/bin\/qemu-aarch64-static\".*/c\#rm -f "${L4T_ROOTFS_DIR}\/usr\/bin\/qemu-aarch64-static\"' ./Linux_for_Tegra/nv_tegra/nv-apply-debs.sh
 
 # In 32.3.1, this will use deb packages instead of the archive overlay
 RUN ./Linux_for_Tegra/apply_binaries.sh -r /

--- a/generation/ubuntu1804/l4t/Dockerfile.jinja
+++ b/generation/ubuntu1804/l4t/Dockerfile.jinja
@@ -53,6 +53,7 @@ RUN sed -i '/.*LC_ALL=C APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=1 chroot . apt-key 
 RUN sed -i '/.*umount ${L4T_ROOTFS_DIR}\/proc.*/c\#umount ${L4T_ROOTFS_DIR}\/proc' ./Linux_for_Tegra/nv_tegra/nv-apply-debs.sh
 RUN sed -i '/.*	LC_ALL=C chroot . dpkg -i --path-include=\"\/usr\/share\/doc\/\*\" \"${pre_deb_list\[\@\]}\".*/c\	apt install -y --no-install-recommends \"${pre_deb_list[@]}\"' ./Linux_for_Tegra/nv_tegra/nv-apply-debs.sh
 RUN sed -i '/.*LC_ALL=C chroot . dpkg -i --path-include=\"\/usr\/share\/doc\/\*\" \"${deb_list\[\@\]}\".*/c\apt install -y --no-install-recommends \"${deb_list[@]}\"' ./Linux_for_Tegra/nv_tegra/nv-apply-debs.sh
+RUN sed -i '/.*rm -f "${L4T_ROOTFS_DIR}\/usr\/bin\/qemu-aarch64-static\".*/c\#rm -f "${L4T_ROOTFS_DIR}\/usr\/bin\/qemu-aarch64-static\"' ./Linux_for_Tegra/nv_tegra/nv-apply-debs.sh
 {% endif %}
 
 {%- if ctx.drivers.version == "32.3.1" %}

